packages/hash/frontend/src/components/hooks/useGetAllEntityTypes.ts

```diff
@@ -5,18 +5,19 @@ import {
 } from "../../graphql/apiTypes.gen";
 import { getAccountEntityTypes } from "../../graphql/queries/account.queries";
 
-export const useAccountEntityTypes = (
-  accountId: string,
-  includeOtherTypesInUse?: boolean,
-) => {
+/**
+ * @todo consolidate this and useBlockProtocolAggregateEntityTypes
```

Comment:
I think we might be able to just outright delete the `useGetAllEntityTypes` hook, assuming `useBlockProtocolAggregateEntityTypes` will let us aggregate the entity types in a users account. Appreciate this is probably what you mean with "consolidate"